code/modules/surgery/organic_steps.dm

```diff
@@ -79,9 +86,18 @@
 /datum/surgery_step/retract_skin/preop(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
 	display_results(user, target, span_notice("You begin to retract the skin in [target]'s [parse_zone(target_zone)]..."),
 		span_notice("[user] begins to retract the skin in [target]'s [parse_zone(target_zone)]."),
-		span_notice("[user] begins to retract the skin in [target]'s [parse_zone(target_zone)]."))
+		span_notice("[user] begins to retract the skin in [target]'s [parse_zone(target_zone)]."),
+		playsound(get_turf(target), 'sound/surgery/retractor1.ogg', 75, TRUE, falloff_exponent = 12, falloff_distance = 1))
 	display_pain(target, "You feel a severe stinging pain spreading across your [parse_zone(target_zone)] as the skin is pulled back!")
 
+/datum/surgery_step/retract_skin/success(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery, default_display_results)
+	display_results(user, target, span_notice("You retract the skin from [target]'s [parse_zone(target_zone)]."),
+		span_notice("[user] retracts the skin from [target]'s [parse_zone(target_zone)]."),
+		span_notice("[user] retracts the skin from [target]'s [parse_zone(target_zone)]."),
+		playsound(get_turf(target), 'sound/surgery/retractor2.ogg', 75, TRUE, falloff_exponent = 12, falloff_distance = 1))
```

Comment:
Why are these all treated as arguments to display_results? This makes no sense, make it its own line.

Reply:
Okay I'll move them over!

Reply:
You could also make the sound files a variable of surgery_step and add the playsound() to initiate() to cut down on copy pasted code.